beets/util/confit.py

```diff
@@ -1242,30 +1248,68 @@ def __init__(self, split=True):
         super(StrSeq, self).__init__()
         self.split = split
 
+    def _convert_value(self, x, view):
```

Comment:
Should this be a static method?

Reply:
I think it's not easily turned into one since it uses `self.fail`. I guess if at a later point the function turns out to be useful elsewhere, the error handling could still be moved to an outer scope?